sktime/detection/base/_base.py

```diff
@@ -493,6 +493,23 @@ def _predict_scores(self, X):
             Labels for sequence X exact format depends on detection type.
         """
         raise NotImplementedError("abstract method")
+    
+    def _transform_scores(self, X):
```

Comment:
This follows the [skchange base class](https://github.com/NorskRegnesentral/skchange/blob/main/skchange/base.py#L283-L300).